app/error.jsx

```
@@ -0,0 +1,33 @@
+"use client";
+
+import Link from "next/link";
+import React from "react";
+
+const Error = ({ error, reset }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)